index.html

```diff
@@ -3,136 +3,328 @@
 
 <head>
   <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-  
-    <!-- Bootstrap CSS -->
-    <link rel="stylesheet"
-        href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
-        integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
-        crossorigin="anonymous">
-    <link rel="stylesheet"
-        href="main.css">
-    <title>QuestBoard
-    </title>
+  <meta name="viewport"
+    content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+  <!-- Bootstrap CSS -->
+  <link rel="stylesheet"
+    href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+    integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
+    crossorigin="anonymous">
+  <link rel="stylesheet"
+    href="main.css">
+  <title>QuestBoard
+  </title>
 </head>
 
 <body>
   <header>
-    <div id="carouselIndicators" class="carousel slide" data-ride="carousel">
-      <ol class="carousel-indicators">
-        <li data-target="#carouselIndicators" data-slide-to="0" class="active"></li>
-        <li data-target="#carouselIndicators" data-slide-to="1"></li>
-        <li data-target="#carouselIndicators" data-slide-to="2"></li>
-        <li data-target="#carouselIndicators" data-slide-to="3"></li>
+    <div
+      id="carouselIndicators"
+      class="carousel slide"
+      data-ride="carousel">
+      <ol
+        class="carousel-indicators">
+        <li
+          data-target="#carouselIndicators"
+          data-slide-to="0"
+          class="active"></li>
+        <li
+          data-target="#carouselIndicators"
+          data-slide-to="1">
+        </li>
+        <li
+          data-target="#carouselIndicators"
+          data-slide-to="2">
```

Comment:
Bad practice - too many enters

Reply:
I think that is not a bad practice - we have to decide if we want to keep attributes in new lines or try to keep it one line. But nice to point it out and talk about that :)